main/lsp/requests/completion.cc

```diff
@@ -63,10 +70,11 @@ const RubyKeyword rubyKeywords[] = {
     {"true", "Boolean true."},
     // This is also defined on Kernel
     // {"undef", "Prevents a class or module from responding to a method call."},
-    {"unless", "Used for unless and modifier unless expressions."},
-    {"until", "Creates a loop that executes until the condition is true."},
-    {"when", "A condition in a case expression."},
-    {"while", "Creates a loop that executes while the condition is true."},
+    {"unless", "Used for unless and modifier unless expressions.", "unless ${1:expr}\n  $0\nend"},
```

Comment:
Can this one fire if you're writing a trailing `unless`? Can this:
```
x = 10 unless
```
expand out to this:
```
x = 10 unless ${1:expr}
end
```

Reply:
Yeah, i punted on `if` vs `post-if` and same for unless... i'm not sure of an easy way around this. I think that people will just learn to not take the snippet

one alternative is to have two snippets, one for post + one for pre, and let the user choose

Reply:
added this to my list.